tests/crypto/mbedtls/testcase.yaml

```diff
@@ -3,3 +3,4 @@ tests:
         min_ram: 32
         tags: crypto mbedtls
         timeout: 200
+        filter: not CONFIG_DEBUG
```

Comment:
should we have a bug opened for the fact that mbedtls has issues with DEBUG (not sure what those issues are)?